PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -12,12 +12,20 @@
   , 'inclusiveSecondaryVertexFinderFilteredNegativeTagInfos'
     # new candidate-based framework (supported with RECO/AOD/MiniAOD)
   , 'pfImpactParameterTagInfos'
+  , 'pfImpactParameterTagInfosAK8'
+  , 'pfImpactParameterTagInfosCA15'
   , 'pfSecondaryVertexTagInfos'
   , 'pfSecondaryVertexNegativeTagInfos'
   , 'pfInclusiveSecondaryVertexFinderTagInfos'
+  , 'pfInclusiveSecondaryVertexFinderTagInfosAK8'
```

Comment:
Why not? There is no attempt to reconstruct SVs so what really matters the most is the kinematics of boosted hadronically decaying reconances. And if we want to capture all IVF SVs belonging to such resonances, it is natural to enlarge the cone size. In addition, this is also beneficial for picking up as many of the displaced tracks as possible. This is quite robust even when there might be other jets (possibly from pileup) overlapping with the fat jet (this was one of the main points ATLAS demonstrated at BOOST2014).